modules/tensor_mechanics/src/userobjects/CrystalPlasticityStateVarRateComponentGSS.C

```diff
@@ -28,6 +30,8 @@ validParams<CrystalPlasticityStateVarRateComponentGSS>()
       "",
       "Name of the file containing the values of hardness evolution parameters");
   params.addParam<std::vector<Real>>("hprops", "Hardening properties");
+  MooseEnum crystal_type("FCC12 BCC12", "FCC12");
+  params.addParam<MooseEnum>("crystal_type", crystal_type, "Crystal structure, default FCC12");
```

Comment:
I'm not a fan of this parameter name because it is too ambiguous; I'd prefer something along the lines of `crystal_lattice_type` or `crystal_lattice_structure_type`

Reply:
this can be corrected quickly. I'll do that when all test are finished.